fdmi/plugins/motr-fdmi-app.md

```diff
@@ -1,33 +1,70 @@
-# Cortx Motr FDMI Application 
+# CORTX-Motr FDMI Applications / Plugins 
 
-FDMI applications can be seen as traditional object or key-value applications that has a more closed relationship with Cortx-Motr. 
+FDMI stands for File Data Manipulation Interface and it is a feature in CORTX that allows developers to add new capabilities and functionalities to the system without having to natively modify the core code. 
 
-In this folder, we have an example of an FDMI application (fdmi_sample_plugin) that will be compiled as part of the initial Motr compilation using the Makefile.am dependency and the Makefile.sub file.
+This extension interface is really powerful since it gives more flexibility to developers so they can easily modify and deploy in CORTX independently on additional and dedicaded nodes without compromising the fast path for Motr. 
 
-To run a simple test of this example, you can execute the following command inside this directory:
+In addition, this interface inherits the horizontal scalability property form motr while being reliable since it will be transactionally coupled with the core.
+  
+FDMI applications can be seen as traditional object or key-value applications that has a more closed relationship with CORTX-Motr. 
 
-`./fdmi_plugin_st.sh`
+# FDMI Implementations Overview
+
+FDMI is a scalable publish-subscribe interface where each Motr instance works as a source and produces records that describe operations. 
+
+On the other side, there is a FDMI Application or Plugin that registers a filter in the Motr Filter Database to selects records that match such filters.
+
+The intial substrings to match metadata fields are setup at the CORTX initialization time.
```

Comment:
The filter substrings to match the `Value` part of the index Key/Value pair are configured at the CORTX initialization time.